docs/01-overview/main-areas/observability/obsv-10-useful-links.md

```diff
@@ -7,10 +7,14 @@ If you're interested in learning more about the Observability area, check out th
 - Learn how to set up the [Monitoring Flow](../../../03-tutorials/00-observability) for your services in Kyma, in which you [observe your application metrics](../../../03-tutorials/00-observability/obsv-01-observe-application-metrics.md), [set up your Grafana Dashboard](../../../03-tutorials/00-observability/obsv-02-create-and-configure-grafana-dashboard.md), [define alerting rules](../../../03-tutorials/00-observability/obsv-03-define-alerting-rules-monitor.md), and eventually, [send notifications](../../../03-tutorials/00-observability/obsv-04-send-notifications.md). And remember to [clean up when you're done](../../../03-tutorials/00-observability/obsv-05-clean-up-configuration.md). 
 
 - Check out the different ways to [access the logs](../../../04-operation-guides/operations/obsv-01-access-logs.md) provided by Kubernetes and Loki.
+- Learn how to adjust Loki's [log limits](../../../04-operation-guides/operations/obsv-02-adjust-loki.md).
 
 - See how to [access and expose](../../../04-operation-guides/security/sec-06-access-expose-kiali-grafana.md) the services Grafana, Jaeger, and Kiali.
 
-- Find out what to do when the [Prometheus Istio Server keeps crashing](../../../04-operation-guides/troubleshooting/obsv-01-troubleshoot-prometheus-istio-server-crash-oom.md) or when [Jaeger shows fewer traces than expected](../../../04-operation-guides/troubleshooting/obsv-02-troubleshoot-jaeger-shows-few-traces.md).
+- Troubleshoot Observability-related issues:
+  - [Prometheus Istio Server keeps crashing](../../../04-operation-guides/troubleshooting/obsv-01-troubleshoot-prometheus-istio-server-crash-oom.md)
+  - [Jaeger shows fewer traces than expected](../../../04-operation-guides/troubleshooting/obsv-02-troubleshoot-jaeger-shows-few-traces.md)
+  - [Loki shows fewer logs than expected](../../../04-operation-guides/troubleshooting/obsv-03-troubleshoot-loki-logging.md)
 
 - Understand the architecture of Kyma's [monitoring](../../../05-technical-reference/00-architecture/obsv-01-architecture-monitoring.md docs/05-technical-reference/00-configuration-parameters/obsv-01-configpara-monitoring.md), [logging](../../../05-technical-reference/00-architecture/obsv-02-architecture-logging.md), and [tracing](../../../05-technical-reference/00-architecture/obsv-03-architecture-tracing.md) components.
```

Comment:
GitHub won't allow me to make a suggestion as it wasn't changed in this PR, but this monitoring link does not display correctly: 

<img width="1259" alt="Screen Shot 2021-12-02 at 13 10 22" src="https://user-images.githubusercontent.com/49867570/144419583-6cb91e62-7bb8-4e25-8444-8f296b5a1097.png">

Reply:
markdown links are the bane of my existence. Thanks for noticing this

Reply:
fixed with https://github.com/kyma-project/kyma/pull/12704/commits/61bd63962d3969ec85cd811ed1e8e64c1aa20bc9